CITATION.cff

```diff
@@ -28,14 +24,18 @@ authors:
   family-names: Grund
   affiliation: Innoplexia GmbH, Germany
   orcid: https://orcid.org/0000-0001-8759-2018
-- given-names: Liam
-  family-names: Toney
-  affiliation: University of Alaska Fairbanks, USA
-  orcid: https://orcid.org/0000-0003-0167-9433
+- given-names: Max
+  family-names: Jones
+  affiliation: University of Hawai'i at Mānoa, USA
```

Comment:
I still have a non-compensated research affiliation with UH, which is more appropriate for my contributions to PyGMT since I do not have time at work to contribute on this project.